index.html

```diff
@@ -582,13 +582,13 @@ <h3>Use Cases and Requirements</h3>
         </p>
 
         <p>
-Digital proof mechanisms, a subset of which are digital signatures, are required
-to ensure the protection of a <a>verifiable credential</a>. Having and
-validating proofs, which may be dependent on the syntax of the proof
+<a>Verifiable credential</a> and <a>verifiable presentation</a> MUST be protected using a digital proof
```

Comment:
This is creating a normative statement in an informative section, which we can't do. If we want to say this normatively, we'll have to say it in a normative section in the specification. You might see if some of this text can be moved here: https://w3c.github.io/vc-data-model/#securing-verifiable-credentials

Reply:
Unless I'm missing something, this is inside the `conformance` section, which is marked as normative. See https://github.com/w3c/vc-data-model/blob/9087738b6a3249b6f3702a522d4aeb183685cda1/index.html#L556

Reply:
I see the problem... 

```
<section id="conformance" class="normative">
``` 
is parallel to (but has no `<h3>`!) 
```
      <section class="informative">
        <h3>Use Cases and Requirements</h3>
```
and both are within
```
    <section class="informative">
      <h2>Introduction</h2>
```
Really, all the 1.x sub-sections should NOT be labeled as non-normative, because they inherit this from 1.0, and there REALLY should be a way to label a section as normative!

Also, the normative `Conformance` section REALLY cannot be within the informative `Introduction`, so the whole section needs to move, not just the sentence seen above on line 585.

Reply:
@TallTed thanks for the explanation.

I'm happy to move sections to more appropriate places, even though that's separate from the scope of the changes intended to be done with this PR. 

Regarding

> Really, all the 1.x sub-sections should NOT be labeled as non-normative, because they inherit this from 1.0, and there REALLY should be a way to label a section as normative!

Labelling the `Introduction` section as normative seems to be what you're suggesting. If not, can you clarify? 


Re:
> Also, the normative Conformance section REALLY cannot be within the informative Introduction, so the whole section needs to move, not just the sentence seen above on line 585.

This depends on the previous point. Assuming that we keep the `Introduction` section as normative, is creating a new entire section appropriate? Where? I fear that moving the whole Conformace section would break the flow of the document, and will need a lot of editorial love.

Reply:
>> [@TallTed] Really, all the 1.x sub-sections should NOT be labeled as non-normative, because they inherit this from 1.0, and there REALLY should be a way to label a section as normative!
>
> [@andresuribe87] Labelling the Introduction section as normative seems to be what you're suggesting. If not, can you clarify?

I wasn't suggesting that, but it's the only way to make the current `Conformance` sub-section normative within the (currently non-normative) `Introduction`, because normative sub-sections are not allowed within non-normative sections.

@iherman may be able to tell us whether `Introduction` can be made normative, such that `Conformance` may remain where it is *and* be made normative, given that all the *other* sub-sections of `Introduction` are already labeled as non-normative.

If `Introduction` must remain non-normative, @msporny can probably point out a good place to move `Conformance` to, such that it can be normative.

Reply:
> Also, the normative Conformance section REALLY cannot be within the informative Introduction, so the whole section needs to move, not just the sentence seen above on line 585.

Err, hold up -- you can have a normative section buried deep in a non-normative section; remember, we got all the way through two RECs in this state. That said, we could also move Conformance up to it's own section... I think I moved it initially because having a top-level section, just for conformance classes, felt like it was a bit much.

I'd prefer to leave the conformance section as it is... which brings me back around to why I objected to this change... @andresuribe87 is indeed correct, the normative statement is added in the conformance section, which is normative, so his initial text is fine and I'm retracting my change request and approving the PR for merge.